max_speed: 10.0
max_steering: 0.6